packages/esbuild-plugin/src/index.ts

```diff
@@ -38,34 +38,42 @@ function esbuildReleaseInjectionPlugin(injectionCode: string): UnpluginOptions {
 
 function esbuildDebugIdInjectionPlugin(): UnpluginOptions {
   const pluginName = "sentry-esbuild-debug-id-injection-plugin";
-  const proxyNamespace = "sentry-debug-id-proxy";
   const stubNamespace = "sentry-debug-id-stub";
 
   return {
     name: pluginName,
 
     esbuild: {
       setup({ onLoad, onResolve }) {
-        onResolve({ filter: /.*/ }, (args) => {
+        onResolve({ filter: /.*/, namespace: "file" }, (args) => {
```

Comment:
do we need this? We are just hard-coding `file` below 😅

Reply:
You are right, the namespace filter here is very likely unnecessary since we only want to resolve entrypoints which _should_ only have the "file" namespace anyways (since "file" is the default).

The "passthrough" namespace below is then also unnecessary --> removed in 5c24b303061dcfb5ab5e73db1eba518b81f948fd